Client/Frontend/UserContent/UserScripts/CookieControl.js

```diff
@@ -0,0 +1,40 @@
+// The $<> is used to replace the var name at runtime with a random string.
+
+//Cookie should neither be saved nor accessed (local or session) when user has blocked all cookies.
+var $<cookie> = Object.getOwnPropertyDescriptor(Document.prototype, 'cookie') ||
```

Comment:
Can you fix up the indentation a bit here? May need to use a 3rd party editor, but would be nice to keep clean.
(I know some of the other JS files aren't nice looking, but just leave those for now).

Reply:
Tried many 3p editor all do the same thing with the standard format settings. 
<img width="1452" alt="screenshot 2019-01-23 at 9 44 09 pm" src="https://user-images.githubusercontent.com/5398557/51620321-1ae04800-1f58-11e9-8157-2da72c94d5d9.png">

Reply:
If this is all, I will push this

Reply:
Done :) Took some trick